SPECS/cloud-init/cloud-init.spec

```diff
@@ -132,21 +131,23 @@ make check %{?_smp_mflags}
 %dir %{_sysconfdir}/cloud/templates
 %doc %{_sysconfdir}/cloud/cloud.cfg.d/README
 %doc %{_sysconfdir}/cloud/clean.d/README
-%{_sysconfdir}/dhcp/dhclient-exit-hooks.d/hook-dhclient
-%{_sysconfdir}/NetworkManager/dispatcher.d/hook-network-manager
 %config(noreplace) %{_sysconfdir}/cloud/templates/*
 %config(noreplace) %{_sysconfdir}/cloud/cloud.cfg
 %config(noreplace) %{_sysconfdir}/cloud/cloud.cfg.d/05_logging.cfg
 %config(noreplace) %{_sysconfdir}/systemd/system/sshd-keygen@.service.d/disable-sshd-keygen-if-cloud-init-active.conf
 %{_unitdir}/*
 %{_systemdgeneratordir}/cloud-init-generator
-/lib/udev/rules.d/66-azure-ephemeral.rules
+/usr/lib/udev/rules.d/66-azure-ephemeral.rules
```

Comment:
Any particular reason for this path change? Is there an upstream change linked to a move?

Reply:
I caught this when I encountered a building error that states this file does not exist. I checked the upstream and saw that there was a recent change on default path of udev/rules (see PR: https://github.com/canonical/cloud-init/commit/70715125f3af118ae242770e61064c24f41e9a02). Change the path from /lib to /usr/lib will resolve this issue.